proton-j/src/main/java/org/apache/qpid/proton/engine/impl/BuiltinEventTypeImpl.java

```diff
@@ -0,0 +1,144 @@
+package org.apache.qpid.proton.engine.impl;
+
+import org.apache.qpid.proton.engine.BuiltinHandler;
+import org.apache.qpid.proton.engine.Event;
+import org.apache.qpid.proton.engine.Handler;
+
+public class BuiltinEventTypeImpl {
```

Comment:
This seems to be dispatcher-specific, might be clearer if we named it so?

Same as before regarding the 'Builtin'.